src/hardware/gameblaster.cpp

```diff
@@ -95,9 +95,10 @@ void GameBlaster::Open(const int port_choice, const std::string &card_choice,
 		constexpr auto cutoff_freq = 6000;
 		channel->ConfigureLowPassFilter(order, cutoff_freq);
 		channel->SetLowPassFilter(FilterState::On);
-	} else {
+
+	} else if (!channel->TryParseAndSetCustomFilter(filter_choice)) {
 		if (filter_choice != "off")
-			LOG_WARNING("%s: Invalid filter setting '%s', using off",
+			LOG_WARNING("%s: Invalid 'cms_filter' value: '%s', using 'off'",
```

Comment:
Nice consistency cleanups.

Reply:
Yeah, and now it's a lot more specific, so the user has a much better chance of fixing what's actually wrong :)